README.md

```diff
@@ -87,7 +87,13 @@ To confirm that the virtual environment has been successfully activated, you can
     # Python 3.10.11   <-- Example output
     ```
 
+#### STEP 3. Install `pyqt5` 
 
+Lastly, we will install the GUI toolkit `pyqt5` which is a requirement for the plugin. Run the following command in your terminal or powershell:
+
+```bash 
+pip install pyqt5
```

Comment:
@yrkim98 this is the inline with napari recommended setup?

Reply:
This should be `pip install "napari[pyqt5]" so that we can ensure that we grab a pyqt5 version that will be compatible with napari. https://napari.org/dev/tutorials/fundamentals/installation.html

Reply:
Done!